tools/idf_monitor.py

```diff
@@ -75,6 +75,7 @@ def __init__(
         print_filter,  # type: str
         make='make',  # type: str
         encrypted=False,  # type: bool
+        no_reset=False,  # type: bool
```

Comment:
This is just an optional suggestion.

If we would have 
```python
reset = True
```
instead then it would be easier to read and understand it than a reverse logic variable.

After that, negation has to be used only at one place:
```python
self.reset = not args.no_reset